tests/run-make/wasm-custom-section/rmake.rs

```diff
@@ -1,15 +1,16 @@
 extern crate run_make_support;
 
-use run_make_support::{out_dir, rustc, wasmparser};
+use run_make_support::{out_dir, wasmparser, Rustc};
 use std::collections::HashMap;
 
 fn main() {
     if std::env::var("TARGET").unwrap() != "wasm32-wasip1" {
         return;
     }
 
-    rustc().arg("foo.rs").arg("--target=wasm32-wasip1").run();
-    rustc().arg("bar.rs").arg("--target=wasm32-wasip1").arg("-Clto").arg("-O").run();
+    Rustc::new().input("foo.rs").target("wasm32-wasip1").run();
```

Comment:
Why `rustc()`->`Rustc::new()`?

Reply:
Previously `rustc()` just called `RustcInvocationBuilder::new()` (which is now `Rustc`), it felt like an really unnecessary wrapper

Reply:
I think it looks nicer